bin/server/cmd/root.go

```diff
@@ -410,10 +412,11 @@ func (m *Main) newEmbeddedDB() (*store.DB, error) {
 
 	// Even when Postgres opens Unix domain socket only for connection, it still requires a port as ID to differentiate different Postgres instances.
 	connCfg := dbdriver.ConnectionConfig{
-		Username: m.profile.pgUser,
-		Password: "",
-		Host:     common.GetPostgresSocketDir(),
-		Port:     fmt.Sprintf("%d", m.profile.datastorePort),
+		Username:    m.profile.pgUser,
```

Comment:
I was thinking this is a good place to create our bytebase database, thus we don't need to pass StrictUseDb to SetupMigrationIfNeeded to create the conditionally create the database.

Do I miss anything?

Reply:
This might provide a consistent model for us supporting a single Postgres database (on render).

Reply:
Ah, creating bb database can be done here, maybe.